seed-element.html

```diff
@@ -1,97 +1,110 @@
 <!--
-    @license
-    Copyright (c) 2014 The Polymer Project Authors. All rights reserved.
-    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-    Code distributed by Google as part of the polymer project is also
-    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <link rel="import" href="../polymer/polymer.html">
 
 <!--
-Element providing solution to no problem in particular.
+An element providing a solution to no problem in particular.
 
-##### Example
+Example:
 
     <seed-element></seed-element>
 
-@element seed-element
-@blurb Element providing solution to no problem in particular.
-@status alpha
-@homepage http://polymerlabs.github.io/seed-element
 -->
-<polymer-element name="seed-element" attributes="notitle author">
+<dom-module id="seed-element">
 
-  <template>
-
-    <link rel="stylesheet" href="seed-element.css">
+  <style>
+    seed-element {
+      display: block;
+      box-sizing: border-box;
```

Comment:
I've always been told to avoid the universal selector. The second one shouldn't be too bad for perf since it's scoped to the element, but I seem to remember the engine first finds all elements, then filters them.  Let's hope that's not still the case :)